### PR TITLE
RegisterNotification: add debugging logging

### PR DESCRIPTION
**Pull request by jendakol**
We need to find out more details about suffering devices.

### CODE REVIEW

File: internal/functions/registernotification/register-notification.go [hunk 1]
```diff
@@ -137,3 +148,17 @@ func handleForFUID(ctx context.Context, fuid string) error {
...
+		return []byte("hello-world"), nil
+	})
+
+	// We certainly got validation error but we don't care. It's just about whether we are able to parse the token or not.
+
+	if token == nil || token.Claims == nil {
+		return 0
+	}
+
+	return int64(token.Claims.(jwt.MapClaims)["exp"].(float64))
```
Comment (marekaf) [inline internal/functions/registernotification/register-notification.go]: is it safe to access `token.Claims.(jwt.MapClaims)["exp"]` without checking for nil?

Comment (jendakol) [reply internal/functions/registernotification/register-notification.go]: Good point... Even though I strongly believe it _will_ be there, it's not safe.

Comment (jendakol) [reply internal/functions/registernotification/register-notification.go]: I'll use the technique shown in [jwt-go](https://github.com/dgrijalva/jwt-go/blob/master/map_claims.go#L22).

File: internal/functions/registernotification/register-notification.go [hunk 2]
```diff
@@ -137,3 +149,28 @@ func handleForFUID(ctx context.Context, fuid string) error {
...
+
+	claims := token.Claims.(jwt.MapClaims)
+
+	switch exp := claims["exp"].(type) {
+	case float64:
+		return int64(exp)
+	case json.Number:
+		v, _ := exp.Int64()
+		return v
+	}
```
Comment (marekaf) [inline internal/functions/registernotification/register-notification.go]: looks much better, but could we maybe add default: with error log "Unknown type %T" or something like that? :D

**Review by marekaf — APPROVED** [review]
lgtm

**Pull request merged**